### PR TITLE
chore: improve code style consistency

### PR DESCRIPTION
**Pull request by smeijer**
## Change description

- The Husky pre-commit hook is now installed during the `prepare` lifecycle. While husky 4 installed itself during `npm i`, husky 7 does not.

- The `lint-staged` config has been fixed by correcting the file patterns in package.json. `*.{tsx?}` doesn't work (anymore?), so that has been expanded to `*.{ts,tsx}`.

- `eslint-plugin-simple-import-sort` has been added so that imports are automatically sorted.

- A few other eslint rules have been enabled, so the code style of this repo matches the style of our Dashboard more closely.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "lint": "tsc --noEmit && eslint '{src,stories,tests}/**/*.{js,ts,tsx}'"
   },
   "lint-staged": {
-    "*.{jsx?,tsx?}": [
+    "*.{js,ts,tsx}": [
```
Comment (josuemontano) [inline package.json:51]: If we ever add `jsx` files these should be included, too. Could we revert this change, please?

**Review by josuemontano — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (smeijer) [reply package.json]: I've updated it to include jsx files.

**Review by josuemontano — APPROVED** [review]
